app/policies/response_template_policy.rb

```diff
@@ -0,0 +1,48 @@
+class ResponseTemplatePolicy < ApplicationPolicy
+  def admin_index?
+    minimal_admin?
+  end
+
+  def moderator_index?
+    user_is_moderator?
+  end
+
+  def create?
+    true
+  end
+
+  # comes from comments_controller
+  def moderator_create?
+    user_is_moderator? && record_is_mod_comment?
+  end
+
+  def destroy?
+    user_is_owner?
+  end
+
+  def update?
+    user_is_owner?
+  end
+
+  def permitted_attributes_for_create
+    %i[content_type content title]
```

Comment:
Can we please pull this out into a constant, especially since it's used in two methods?